packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -1139,6 +1139,17 @@ new ecs.Ec2Service(this, 'EC2Service', {
 });
 ```
 
+### Cluster Default Provider Strategy
+When the service does not have a capacity provider strategy, cluster's default capacity provider strategy will be used.
+```ts
+new ecs.Cluster(stack, 'EC2CPCluster', {
+  vpc,
+  defaultCapacityProviderStrategy: [
+    { capacityProvider: capacityProvider.capacityProviderName, base: 1, weight: 1 },
```

Comment:
Let's use a more complicated default capacity provider strategy. 

For example, let's use a combination of a custom capacity provider and the Fargate Spot provider.